num2words/lang_NL.py

```diff
@@ -129,12 +134,17 @@ def to_ordinal_num(self, value):
         self.verify_ordinal(value)
         return str(value) + "."
 
-    def to_currency(self, val, longval=True, old=False):
-        if old:
-            return self.to_splitnum(val, hightxt="euro/s", lowtxt="cent/s",
-                                    jointxt="en", longval=longval)
-        return super(Num2Word_NL, self).to_currency(val, jointxt="en",
-                                                    longval=longval)
+    def pluralize(self, n, forms):
+        """
+        :param n:
+        :param forms:
+        :return:
+
+        gettext form is nplurals=2; plural=(n != 1);
+        but this claims https://onzetaal.nl/taaladvies/euro-euros/
+        not sure if it's applied only to euro
+        """
+        return forms[0]
```

Comment:
I can't find where is used this method, except that on the test. Do we need it?

Reply:
It's called in to_currency of parent, it's only there because it got pointed out in here #159 .
I'm don't speak Dutch so it's meant more or less as placeholder for when someone implements other other currencies